tests/sentry/notifications/utils/test_participants.py

```diff
@@ -672,9 +672,70 @@ def test_fallthrough_all_members_multiple_teams(self):
         }
 
     @with_feature("organizations:issue-alert-fallback-targeting")
-    def test_fallthrough_admin_or_recent(self):
+    def test_fallthrough_admin_or_recent_under_20(self):
```

Comment:
I want to add a test that ensures we correctly identify the most recently active users. Will push that commit soon.